tests/acceptance/features/apiActivities/activitiesByFileId.feature

```diff
@@ -472,3 +472,809 @@ Feature: check activities
         }
       }
       """
+
+
+  Scenario: check activities of destination folder after moving a file into it using file-id
+    Given user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt"
+    And we save it into "FILEID"
+    And user "Alice" has created folder "FOLDER"
+    And user "Alice" has moved file "textfile.txt" into "FOLDER" inside space "Personal" using file-id "<<FILEID>>"
```

Comment:
how about these steps?
```feature
    And user "Alice" has (copied|moved) a file with id "<<FILEID>>" as "textfile.txt" into folder "FOLDER" of space "Personal"
```

Reply:
Sounds awesome. Leaning to this idea then :slightly_smiling_face: